### PR TITLE
fix: Only enforce min seats for orgs

### PR DESCRIPTION
**Pull request by joeauyeung**
## What does this PR do?

This PR fixes the logic to only change team subscription amounts for orgs and not normal teams

Fixes # (issue)

## Requirement/Documentation

- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- Enable team billing
- Create a team and invite members
- Delete some members
- On Stripe the subscription count should be updated

## Mandatory Tasks

- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by zomars — CHANGES_REQUESTED** [review]
We have done types failing

**Review by zomars — CHANGES_REQUESTED** [review]
Some Qs, I'm trying to understand the why of this.

- Where does the ORGANIZATION_MIN_SEATS number coming from?
- Will this affect non orgs teams sub update logic? How can we verify this?
- Is this related to the previous discussion where repeated members of different teams of the same org should not be charged?

File: packages/lib/constants.ts [hunk 1]
```diff
@@ -90,3 +90,5 @@ export const MINUTES_TO_BOOK = process.env.NEXT_PUBLIC_MINUTES_TO_BOOK || "5";
 // Needed for orgs
 export const ALLOWED_HOSTNAMES = JSON.parse(`[${process.env.ALLOWED_HOSTNAMES || ""}]`) as string[];
 export const RESERVED_SUBDOMAINS = JSON.parse(`[${process.env.RESERVED_SUBDOMAINS || ""}]`) as string[];
+
+export const ORGANIZATION_MIN_SEATS = 30;
```
Comment (alannnc) [inline packages/lib/constants.ts:94]: I think we can move this to it's own file? Unless it's being used multiple times on several files.

**Review by alannnc — APPROVED** [review]
Looks good, left a NIT comment.

File: packages/lib/constants.ts (hunk 1 above)
Comment (joeauyeung) [reply packages/lib/constants.ts:94]: Do you have a suggestion on where to move it? I figured the constants file would be an easy place to find this variable.

**Comment by joeauyeung** [comment]
> Some Qs, I'm trying to understand the why of this.
> 
>     * Where does the ORGANIZATION_MIN_SEATS number coming from?
> 
>     * Will this affect non orgs teams sub update logic? How can we verify this?
> 
>     * Is this related to the previous discussion where repeated members of different teams of the same org should not be charged?

- The minimum number is coming from @shirazdole
- The problem was this logic before was affecting non-org teams. I tested this locally and checked the test Stripe dashboard, and the number of seats the customer was being charged is being updated with this fix. Regular teams will have normal members and not orgMembers
- That logic is handled somewhere else. This is only for removing team members

**Review by zomars — APPROVED** [review]
Thanks fro clarifying. LGTM

LGTM

**Pull request merged**